bin/apisix

```diff
@@ -631,7 +631,7 @@ local function init_etcd(show_output)
                 break
             end
 
-            if show_output then
+            if show_output and show_output ~= "-c" then
```

Comment:
It would be better to pass the argument explicitly. Filter out all the other options is error-prone. For example, is `--config` handled?

Reply:
I think we could remove `show_output`, since it's used to debug init_etcd. I raised an issue in #2624